charts/alfresco-ooi-service/templates/ingress-ooi-service.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.ooi.enabled }}
+{{- $serviceName :=  printf "%s-%s" (include "content-services.shortname" .) "ooi-service" -}}
```

Comment:
use `include "content-services.shortname" .`

get rid of any other `content-services.shortname` usage